frontend/views/containers/sidebar/GroupsList.vue

```diff
@@ -1,60 +1,33 @@
 <template lang='pug'>
-div(v-if='groupsByName.length')
-  menu-parent(@select='handleMenuSelect')
-    menu-trigger.c-trigger(:class="`has-text-${isDarkTheme ? 'white' : 'dark'}`")
-      avatar.c-avatar(
-        src='/assets/images/default-avatar.png'
-        :alt='groupSettings.groupName'
-        :blobURL='groupSettings.groupPicture'
-      )
-      | {{ groupSettings.groupName }}
-      i.icon-angle-down
+ul.c-group-list(v-if='groupsByName.length')
+  li.c-group-list-item.group-badge(
+    v-for='(group, index) in groupsByName'
+    :key='`group-${index}`'
+    tag='button'
+    :class="{ 'is-active': currentGroupId === group.contractID}"
+  )
+    avatar.c-avatar(
+      src='/assets/images/default-avatar.png'
```

Comment:
> Another thing I noticed is that the sidebar does not show the group avatar I uploaded. Is this also supposed to be done in another PR?

It must be a bug @mmbotelho yes, here's the cause of the problem: The source image is a static one at the moment. It's a easy fix for @pieer ;)

Reply:
Ths static file is for the default in case there is no image. I believe the avatar is another issue that you're working on?

Reply:
I'm working on the avatar issue but it's on profile avatar, not group avatar. It's different things :)

Reply:
Is it?